### PR TITLE
Update seed files

### PR DESCRIPTION
**Pull request by fivetran-avinash**
**Are you a current Fivetran customer?** 

**What change(s) does this PR introduce?** 

**Did you update the CHANGELOG?** 

[NA] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

[NA] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [x] No 

**How did you test the PR changes?** 

 

- [ ] Buildkite  
- [x] Local (please provide additional testing details below)
Verified integration tests passed locally.

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🛩️ 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -29,10 +29,8 @@ vars:
     jira_user_identifier: "user"
     jira_version_identifier: "version"
 
-    jira_schema: jira_source_integration_tests
-
 seeds:
-  jira_source_integration_tests:
```
Comment (fivetran-joemarkiewicz) [inline integration_tests/dbt_project.yml:35]: Why was this renamed? I am pretty sure this should match the same name as line 1 in this file. The name of the project as opposed to the name of the schema variable.

Comment (fivetran-joemarkiewicz) [reply integration_tests/dbt_project.yml:35]: You can actually see in the BuildKite job that the renamed of the seeds config is causing the job to not recognize the seed files and is not picking them up.
image

**Review by fivetran-joemarkiewicz — COMMENTED** [review]
Just one question on why the seeds config was renamed as it looks like it is now causing unknown path configurations in the runs for buildkite. This should be swapped back. Let me know if it doesn't make sense why this should not be renamed.

Additionally, for future reference you don't need to regen the docs if you are not making any changes to code or root project yml definitions. Since you are just editing seed files and the integration tests, the docs don't need to be regenerated. They will be the same as the previous build.

File: integration_tests/dbt_project.yml (hunk 1 above)
Comment (fivetran-avinash) [reply integration_tests/dbt_project.yml:35]: That was a mistake on my part, forgot to run locally and validate. Reverted.
